src/helpers/blood-pressure-data-providers/index.ts

```diff
@@ -1,3 +1,4 @@
-﻿export { default as bloodPressureDataProvider } from "./survey-blood-pressure-data-provider"
+﻿export { default as bloodPressureDataProvider } from "./combined-blood-pressure-data-provider"
 export { SurveyBloodPressureDataParameters as SurveyBloodPressureDataParameters } from "./survey-blood-pressure-data-provider"
 export { BloodPressureDataPoint as BloodPressureDataPoint } from "./survey-blood-pressure-data-provider"
+export { BloodPressureDeviceDataSource as BloodPressureDeviceDataSource } from "./combined-blood-pressure-data-provider"
```

Comment:
I think you can just add `, BloodPressureDeviceDataSource` to the existing import above.